apps/showcase/src/assets/trainings/sdk/steps/introduction/instructions.html

```diff
@@ -0,0 +1,41 @@
+<h3>Why an SDK?</h3>
+<p>
+  If an application contains an API, there are several developer tasks that are required including a lot of redundant code.
+  These tasks include:
+</p>
+<ul>
+  <li>Choosing the needed techniques to perform API calls</li>
+  <li>Determining which HTTP method to use for each request</li>
+  <li>Managing the Authorization (maybe adding request headers) and do it for each request</li>
+  <li>Preparing the URL with any path or query parameters expected by the API endpoint</li>
+</ul>
+<p>
+  Let's see how we use the OpenAPI tools to handle our APIs and generate a typescript SDK codebase.
+  Check out the following diagram to visualize the description below.
+</p>
+<figure class="figure">
+  <img src="/assets/sdk-training-openapi-diagram.svg" width="75%" class="figure-img img-fluid mx-auto d-block">
+</figure>
+<p>
+  The specifications of the SDK are written in an OpenAPI document, which defines or describes an API and its elements.
+  The API SDK is then generated using the OpenAPI Generator and the Otter SDK templates. These template files include the
```

Comment:
Do we want to detail our implementation in the introduction of our training?

Reply:
It is not really detailed, it's supposed to be a general introduction to explain how our implementation works (paired with the diagram to be able to visualize) which I think is legitimate in an introduction... What does everyone think?